openhcl/underhill_mem/src/lib.rs

```diff
@@ -329,7 +327,16 @@ impl MemoryAcceptor {
                         vtl: vtl.into(),
                     })
             }
+        };
+
+        if let Err(e) = &r {
+            tracelimit::error_ratelimited!(
```

Comment:
In general we shouldn't trace errors that we're returning, unless we're losing information (mapping to a status code or something). The caller (or caller's caller, etc.) should trace if it wants to.

Reply:
If none of the callers can use the error information, then consider tracing and then returning a simpler error type (even a ZST).